shared/Microsoft.AspNetCore.BenchmarkRunner.Sources/AspNetCoreBenchmarkAttribute.cs

```diff
@@ -28,6 +28,7 @@ public AspNetCoreBenchmarkAttribute(Type configType, Type validationConfigType)
                 { NamedConfiguration.Validation, typeof(DefaultCoreValidationConfig) },
                 { NamedConfiguration.Profile, typeof(DefaultCoreProfileConfig) },
                 { NamedConfiguration.Debug, typeof(DefaultCoreDebugConfig) },
+                { NamedConfiguration.Benchmarks, typeof(DefaultCoreBenchmarksConfig) },
```

Comment:
Does it make sense to call this 'lab' or 'perflab' or something more distinct? bechmarks is overloaded out the wazoo

Reply:
Agreed, to many things called benchmarks